### PR TITLE
Retrieved planes from server

### PR DESCRIPTION
**Pull request by nikojohnarellano**

### CODE REVIEW

File: application/config/autoload.php [hunk 1]
```diff
@@ -132,5 +132,5 @@
 |
 |	$autoload['model'] = array('first_model' => 'first');
 */
-$autoload['model'] = array('flight', 'plane', 'flightsList', 'planesList');
+$autoload['model'] = array('flight', 'plane', 'flightsList', 'planesList', 'wackymodel');
```
Comment (danielcapacio) [inline application/config/autoload.php]: Think this should be `wackyModel` in order to access the `WackyModel` model.

Comment (nikojohnarellano) [reply application/config/autoload.php]: its working for me

Comment (nikojohnarellano) [reply application/config/autoload.php]: but can you pull and try just to make sure

Comment (danielcapacio) [reply application/config/autoload.php]: Just tested - app seems to be working fine. However, the travis build is complaining.
wackymodel

Comment (nikojohnarellano) [reply application/config/autoload.php]: I see i guess its needed to be changed for the travis build

**Comment by danielcapacio** [comment]
Excellent!

**Review by danielcapacio — APPROVED** [review]

**Pull request merged**